README.Rmd

```diff
@@ -55,6 +55,23 @@ Otherwise, for a lightweight installation where dependencies for plotting and th
 remotes::install_github("traitecoevo/austraits", upgrade = "ask")
 ```
 
```

Comment:
Perhaps a subheading here "AusTraits versions and backwards compatibility with {austraits}" or something alike? So it's clear this section is seperate from installing the latest version

Reply:
done - I created a subheading, 1 indent in from generic `Installation`